### PR TITLE
 Add unit command bar to RA

### PR DESCRIPTION
**Pull request by pchote**
#13442 / #3789 for RA, plus some minor related bugfixes.

rabar

### CODE REVIEW

**Comment by rob-v** [comment]
Minelayer is special case re Force attack - Button disabled, but Ctrl key works. When more units selected, button then also works for minelayer (as it is enabled and Ctrl works for minelayer).

**Comment by rob-v** [comment]
Chrono tank - after right click on unit, there is debug message: 'BUG: in order targeter....'. It works fine, but this message isn't displayed in release version.

**Comment by rob-v** [comment]
Small note - in case we'll have more special actions on e.g. Force attack, we might need different tooltips for factions, e.g. currently 'Chrono tanks' are in tooltip for all factions (otherwise the tooltip will be long and less relevant).

File: OpenRA.Mods.Cnc/Traits/PortableChrono.cs [hunk 1]
```diff
@@ -90,17 +92,24 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
...
+			// Teleport as close as possible
+			if (CanTeleport)
 			{
 				var maxDistance = Info.HasDistanceLimit ? Info.MaxDistance : (int?)null;
 				self.CancelActivity();
 				self.QueueActivity(new Teleport(self, order.TargetLocation, maxDistance, Info.KillCargo, Info.FlashScreen, Info.ChronoshiftSound));
 			}
+
+			// Move normally the rest of the way
+			move.ResolveOrder(self, new Order("Move", self, true) { TargetLocation = order.TargetLocation });
```
Comment (rob-v) [inline OpenRA.Mods.Cnc/Traits/PortableChrono.cs]: According to ctor, move can be null. Can it happen in practice? Then a null check is misisng.

**Comment by pchote** [comment]
Removed the extra minelayer and chronotank fixes.  Unhacking those traits properly is *way* out of scope of what I want to fix here.

**Review by rob-v — APPROVED** [review]
👍

**Review by atlimit8 — APPROVED** [review]

**Comment by atlimit8** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/74143e9b77b0e84b874e1b351f213ae92e44b8f2...5b34f717e9b1f61937ab12f306755960851445ed)

**Pull request merged**